docs/how-tos/production_configuration.rst

```diff
@@ -0,0 +1,245 @@
+- Can support advanced use cases for sending the xAPI events to other sinks with the same near real-time delivery guarantees as Aspects
+- (redis) Can scale down to use the same redis instance as edx-platform, or use a separate / hosted redis for better isolation and performance
+- (Kafka) Can scale up to handle extremely high volumes of data and handle long-term outages better than any other option, can be run hosted
+- (Kafka) Can potentially `integrate directly`_ with ClickHouse removing the need for both a separate event consumer and Ralph.
+
+Cons:
+
+- Running consumers is not currently handled automatically by Aspects
+- Many parts are new and may not have extensive production testing
+
+Recommended for:
+
+Large-to-very-large instances, adventurous site operators, installations that already have Kafka or advanced use cases that can benefit from a multi-consumer architecture.
+
+
+Setting up the xAPI Pipeline
+############################
+
+**Celery**
+
+When in doubt, the simplest place to start with a production configuration is Celery tasks with batching set to 100 events or 5 seconds. You will want to add at least one additional lms-worker to handle the additional load of xAPI events and the event sink pipeline. You will also probably want to add at least one additional cms-worker to handle the new work of the course publishing event sink.
+
+**Vector**
+
+Generally the Aspects created Vector configuration should work in most cases. In Kubernetes environments you will need to make sure that a Vector pod is attached to each LMS/CMS worker.
+
+**Event bus**
+
+Similar to Celery, you should start with at least 2 event bus consumers and configure batching to 100 events or 5 seconds to start with. If you find that the event queue size is growing (see "Monitoring", below), you can add more event bus consumers and/or increase the batch size. We have tested with batch sizes up to 1000 without issue.
+
+
+Choosing ClickHouse Hosting
+###########################
+
+By default Aspects deploys a single ClickHouse Docker container as part of the Tutor install. This is not the preferred way to run a production environment! In most cases, if you can afford it, ClickHouse Cloud is the easiest and most performant way to run the service, and removes the burden of dealing with scaling, security, upgrades, backups, and other potentially difficult database management issues. Axim has been using ClickHouse Cloud for load testing and is designed to work with it.
+
+Altinity Cloud is another hosting service that Aspects has tested with in the past, but may require more hands-on integration as they use a different clustering approach than ClickHouse Cloud.
+
+Another option if you are running in Kubernetes is to use the `clickhouse-operator`_ to deploy and manage a more fault tolerant ClickHouse cluster. Aspects support for ClickHouse clusters is currently experimental, and may not support all cluster configurations without modification.
```

Comment:
There is another alternative that is to run hosted ClickHouse in a different machine or **native installation`**.

Reply:
This is a good alternative for small to medium-sized deployments or installations that have no idea what the workload would be.

Reply:
Do you have a suggestion for how to communicate those alternatives here?

Reply:
```suggestion
Another option if you are running in Kubernetes is to use the `clickhouse-operator`_ to deploy and manage a more fault tolerant ClickHouse cluster. Aspects support for ClickHouse clusters is currently experimental, and may not support all cluster configurations without modification.

One last deployment option is to install ClickHouse in an independent VM from the Kubernetes cluster, we refer to it as `native installation`. Useful from small to medium-sized installations. See clickhouse installation for more information. Make sure to install the same version as tutor-contrib-aspects and setup the server.xml template as the plugin.
```

Maybe something like this would be enough, there aren't any more details to it